packages/i18n/scripts/build.ts

```diff
@@ -21,7 +22,7 @@ console.time('build');
 // Create languages array
 // Note: The language file `en` does not need to be added as the default
 // messages of Valibot are already in English
-const languages = [de, fr, ja, kr, nb, nl, pt, ro, ru, sl, uk, zhCN, zhTW];
+const languages = [de, fr, ja, kr, nb, nl, pt, ro, ru, sl, tur, uk, zhCN, zhTW];
```

Comment:
@KutayGuler isn't is should be tr why tur?

Reply:
@behramcelen You are right, I got the wrong ISO code